packages/cli/src/commands/doctor/healthchecks/androidSDK.ts

```diff
@@ -47,11 +47,15 @@ export default {
       } catch {}
     }
 
+    const version = sdks === 'Not Found' ? sdks : sdks['Build Tools'][0];
```

Comment:
What if I forgot to install `build-tools` when using `sdkmanager` manually? Here will throw an `Cannot read property '0' of undefined` error, instead of telling me to install `build-tools` through `sdymanager "build-tools;29.0.0"`.